scripts/argparser.lua

```diff
@@ -0,0 +1,20 @@
+local argparse = require("argparse")
+
+argparser = argparse("stormweaver", "")
+argparser:argument("scenario", "Scenario file")
+argparser:option("-c --config", "Configuration file.", "config/stormweaver.toml")
+argparser:option(
+	"-i --install_dir",
+	"PostgreSQL installation directory (if specified, overrides configuration value for default server)",
+	""
+)
```

Comment:
Not a blocker, just wondering, does it makes sense to have an option for datadir_root as well?

Reply:
We might add something like that in the future, but it is less critical than the installation path. It's a corner case if you want to run the tests on a different disk or something.